src/Symfony/Component/Console/Application.php

```diff
@@ -466,7 +466,9 @@ public function get($name)
             $command = $this->commands[$name];
         } elseif ($this->commandLoader && $this->commandLoader->has($name)) {
             $command = $this->commandLoader->get($name);
-            $command->setName($name);
+            if (!$command->getName()) {
```

Comment:
This can be called with an `alias` when the command is lazy, we shouldn't alter the command name in this case. The container is responsible for building the command including setting its name and aliases, hence doing this from the passe. I'm even asking myself if we shouldn't throw if `!$command->getName()` rather than letting the command being named by an alias and have some WTF moments